components/tidb_query/src/expr/builtin_arithmetic.rs

```diff
@@ -537,12 +537,12 @@ mod tests {
             let rhs = datum_expr(tt.2);
 
             let lus = lhs
-                .get_field_type()
+                .get_type()
```

Comment:
It is called `field_type` instead of `type`, so no need to change them.

Reply:
Ah, sorry, could you wait until I fix the CI errors before reviewing? There's a few like this that I didn't catch locally.

Reply:
OK, updated with a version that should be OK now